app/audit.go

```diff
@@ -85,9 +85,9 @@ func (a *App) LogAuditRecWithLevel(rec *audit.Record, level mlog.LogLevel, err e
 // MakeAuditRecord creates a audit record pre-populated with defaults.
 func (a *App) MakeAuditRecord(event string, initialStatus string) *audit.Record {
 	var userID string
-	user, err := user.Current()
+	currentUser, err := user.Current()
```

Comment:
I am wondering whether this is needed? `user` seems readable enough to me.

Reply:
there is import "os/user", that is imho not a good idea

Reply:
I agree that it can indeed be a bit confusing. However, in the context of the code, which is just 3 lines, I think it does not cause that much of a readability issue. The user can immediately see that `user.Uid` and `user.Username` come from the variable above. If it was quite high up in the code, then certainly it would make sense.

What do you think?

Reply:
I'm just following suggestion from the jetbrains knowledge (and i think, this community has a reason for this suggestion)  and the clean code rules of uncle Bob. I still think, `currentUser` is better identifier than `user`. But that is not my own code, i'm not the maintainer, so i've reverted the change.

Reply:
IMO `currentUser` makes complete sense if the variable was used in a long method/function. However, for just 3 lines of code, Go prefers to have shorter variable names.

To be more pedantic, the length of the variable name should be directly proportional to the longest distance between its instantiation and usage. https://github.com/golang/go/wiki/CodeReviewComments#variable-names.

Given that, ideally it should have been `usr`. But considering the fact that the code is already written, and every change has a cost, IMO changing it wouldn't be worth the cost it would have.

Reply:
i must say, the part with the variable names is new for me. But my own coding style is a little bit different. I still preffer longer variable names instad of `i` or `c`. But that is just my coding style :) everything is fine for me!